lib/passport-wsfed-saml2/saml.js

```diff
@@ -322,14 +354,11 @@ SAML.prototype.parseAssertion = function(samlAssertion, callback) {
   if (!samlAssertion.getAttribute)
     samlAssertion = samlAssertion.documentElement;
 
-  var version;
-  if (samlAssertion.getAttribute('MajorVersion') === '1')
-    version = '1.1';
-  else if (samlAssertion.getAttribute('Version') === '2.0')
-    version = '2.0';
-  else
+  var version = utils.getSamlAssertionVersion(samlAssertion);
+  let supported_versions = ['1.1', '2.0'];
```

Comment:
This should be a `const` and it looks like the rest of the file mostly uses `camelCase` so I'd recommend this var match.